test/TensorFlow/integration.swift

```diff
@@ -495,12 +495,9 @@ public func testNoInlineUser() {
 // CHECK: [[XS:%.*]] = struct $Tensor<Float> ([[X]] : $TensorHandle<Float>)
 // CHECK:  strong_retain [[SOME_T:%.*]] : $TensorHandle<Float>
 // CHECK-NEXT:  strong_release [[SOME_T]] : $TensorHandle<Float>
-// CHECK:  strong_retain [[SOME_T]] : $TensorHandle<Float>
```

Comment:
I am guessing that these differences are caused by the switch to guaranteed calling, but could someone confirm that this is correct?

Reply:
Can you provide the full SIL code?

Reply:
here's the SIL for the TFPartition Host Result for this function: https://gist.github.com/marcrasi/b2fdaeb542b9b9d307bb49efe14e1406

I also included the TFPartition Host Result for the function that I changed in retain_release.swift, in case you want to look at that.